src/Components/ArticlePage.jsx

```diff
@@ -1,25 +1,47 @@
 import Card from 'react-bootstrap/Card'
 import Button from 'react-bootstrap/Button'
-import {useParams} from 'react-router'
-import {Link} from 'react-router'
+import {useParams, Link } from 'react-router'
+import {useEffect, useState}from 'react'
+import axios from 'axios'
+import CommentCard from './CommentCard'
 
-
-function ArticlePage ({articles}){
+function ArticlePage (){
     const {article_id} = useParams();
-    const article = articles.find((article) => article.article_id.toString() === article_id);
+    const [article, setArticle] = useState([])
+    const [comments, setComments] = useState([])
+    useEffect(() => {
+        const getArticleById = async () => {
+            const { data } = await axios.get(`https://nc-news-lo7q.onrender.com/api/articles/${article_id}`)
+            setArticle(data.article)
+        }
+        getArticleById()
+    }, [article_id])
+
+    useEffect(() => {
+        const getCommentsByArticleId= async () => {
+            const { data } = await axios.get(`https://nc-news-lo7q.onrender.com/api/articles/${article_id}/comments`)
+            setComments(data.comments)
```

Comment:
If what we get back is an empty array, which in this case would be for article with the id of 37, then the user should be told that no comments exist in some way